snapshot/src/test/java/io/zeebe/snapshots/broker/impl/FileBasedTransientSnapshotTest.java

```diff
@@ -40,7 +30,7 @@
 public class FileBasedTransientSnapshotTest {
 
   @Rule public TemporaryFolder temporaryFolder = new TemporaryFolder();
-  private PersistedSnapshotStore persistedSnapshotStore;
+  private FileBasedSnapshotStore persistedSnapshotStore;
```

Comment:
i would expect that we still can use the interfaces otherwise we did something wrong in our api